src/coord/coord.rs

```diff
@@ -765,6 +765,7 @@ where
                 };
                 match self.catalog_transact(vec![op]) {
                     Ok(()) => {
+                        self.catalog.increment_sink_count();
```

Comment:
this doesn't properly account for what happens if you brought back a sink stored in the catalog